base_layer/core/src/blocks/genesis_block.rs

```diff
@@ -399,11 +393,13 @@ mod test {
     }
 
     #[test]
-    fn esmeralda_genesis_sanity_check() {
```

Comment:
This esemeralda genesis block sanity test should not be removed, it should be added back in. We need to do the genesis block sanity check for every network.